cameras:
  - name: cs050
    serial_number: "DA6320570"
    exposure:
      auto: true
      min: 15   # 自动曝光下限
      max: 13319  # 自动曝光上限
      mode: 0             # 0: Timed, 1: TriggerWidth
      value: 5000       # 曝光时间（us）
    gain:
      auto: true         # 是否自动增益
      min: 0.0  # 自动增益下限
      max: 22.0 # 自动增益上限
      value: 6.0       # 增益值 (dB)
    white_balance:
      auto: false
    roi:
      width: 2448
      height: 2048
      offset_x: 0
      offset_y: 0

loop_rate_hz: 200
